src/graph/graphcommandexecutor.pas

```diff
@@ -127,6 +145,8 @@ procedure TGraphCommandExecutor.UpdateChartTitles(ST: TCustomGraphCommand;
     if (inc > 0) then
       Chart.Title.Font.Size := Chart.Title.Font.Size + inc;
 
+    ST.HasOption(['s', 'save', 'e', 'export'], Opt);
+
     ST.HasOption(['footer', 'fn'], Opt);
```

Comment:
This line is prone to same error as above regarding the **Opt** being a null pointer.

Reply:
see comment above